project-base/storefront/pages/order-confirmation.tsx

```diff
@@ -32,32 +32,33 @@ export type OrderConfirmationQuery = {
 const OrderConfirmationPage: FC<ServerSidePropsType> = () => {
     const { t } = useTranslation();
     const { query } = useRouter();
-    const { fetchCart, isWithCart } = useCurrentCart(false);
+    const { fetchCart } = useCurrentCart(false);
     const { orderUuid, orderPaymentType } = query as OrderConfirmationQuery;
 
     const gtmStaticPageViewEvent = useGtmStaticPageViewEvent(GtmPageType.order_confirmation);
     useGtmPageViewEvent(gtmStaticPageViewEvent);
 
-    const [{ data: orderSentPageContentData, fetching }] = useOrderSentPageContentQuery({
+    const [{ data: orderSentPageContentData, fetching: isFetching }] = useOrderSentPageContentQuery({
```

Comment:
Please rename isFetching to a more specific name, it would help to read the code somewhere below

Reply:
This has been rewritten everywhere.